earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -52,6 +54,15 @@
 # Minimum portable globe size in MB.
 GLOBE_SIZE_THRESHOLD = 1.0
 
+# Redirect Earth Client requests with no Globe specified on the URL
+# to the default_ge based URIs to provide Earth Client access to a default map 
+# if none is specified in the server connection URLs. 
+EC_DEFAULT_MAP_LINE0_LOCAL_REWRITECOND =  "RewriteCond %{HTTP_USER_AGENT}  ^EarthClient/(.*)$\n"
+EC_DEFAULT_MAP_LINE1_GOOGLE_REDIRECT = (
```

Comment:
Style: I think templates with named variables are better.  They don't require knowing the variables and their order to expand.

Reply:
I agree - but wanted to remain consistent with the existing templates, rather than refactor them, which would involve much more testing.

Reply:
I wasn't suggesting to refactor the existing ones, but you can make yours more readable.